single/docker-compose.yml

```diff
@@ -4,9 +4,10 @@ services:
     image: nutsfoundation/nuts-node:master
     environment:
       NUTS_CONFIGFILE: /opt/nuts/nuts.yaml
+      NUTS_NETWORK_BOOTSTRAPNODES: ""
     ports:
       - "127.0.0.1:1323:1323"
```

Comment:
this one as well for `/n2n` and the access token

Reply:
Ouch, that means all administrative operations are exposed as well... Ofc not a possibility for a "data leak" with production data, but maybe a bit sloppy?

We could bind `/n2n` to a different port...?

Reply:
Doesn't really matter for demo purposes. In production your reverse proxy will require client certs for that interface/path.